openquake/calculators/calc.py

```diff
@@ -221,7 +221,20 @@ def gmvs_to_haz_curve(gmvs, imls, invest_time, duration):
 
 # ################## utilities for classical calculators ################ #
 
-def make_uhs(maps):
+def get_imts_periods(imtls):
+    """
+    Returns a list of IMT strings and a list of periods. There is an element
+    for each IMT of type Spectral Acceleration (PGA is considered an alias
+    for SA(0.0))
+
+    :param imtls: as set of intensity measure type strings
+    """
+    imts = sorted(from_string(imt) for imt in imtls
+                  if imt.startswith('SA') or imt == 'PGA')
+    return map(str, imts), [imt[1] or 0.0 for imt in imts]
```

Comment:
It looks like this is returning something different with respect to what described in the docstring